.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
+    directory: "/"
+    schedule:
+      interval: daily
+    reviewers:
+      - fastly/developer-relations
```

Comment:
Actually, I think this is going to cause problems when this file inevitably ends up in customer repositories.

Reply:
I just ran `fastly compute init` and created a project using the default Rust starter kit and the `.github` directory was not present, so I suspect at least the CLI is already taking care of this.